cves/CVE-2011-0698.yml

```diff
@@ -167,8 +191,8 @@ subsystem:
     In the name field, a subsystem name (or an array of names)
 
     e.g. clipboard, model, view, controller, mod_dav, ui, authentication
-  answer:
-  name:
+  answer: The file path of the file that was changed was  django/contrib/sessions/backends/file.py
+  name: Sessions-Backend
```

Comment:
I believe it is session-storage https://www.djangoproject.com/weblog/2011/feb/08/security/